parcer-for-log2/picnic_list.json

```diff
@@ -0,0 +1,17 @@
+{
+ "data_per_oreah": 
```

Comment:
The name is misleading, the data under this is not per a person.
Perhaps you should call it "food_and_drinks".